x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -179,8 +192,9 @@ const GraphOverlayComponent = ({
 
       <EuiHorizontalRule margin="none" />
       <StyledResolver
-        databaseDocumentID={graphEventId}
+        databaseDocumentID={graphEventId ?? ''}
```

Comment:
@oatkiller should we have the backend validate that the `_id` not be empty?
https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/endpoint/schema/resolver.ts#L91